GameServerLib/Logic/Config.cs

```diff
@@ -15,7 +15,7 @@ public class Config
         public ContentManager ContentManager { get; private set; }
         public const string VERSION_STRING = "Version 4.20.0.315 [PUBLIC]";
         public static readonly Version VERSION = new Version(4, 20, 0, 315);
-        public const uint VERSION_NUMBER = 42000315;
+        public static readonly uint[] VERSION_NUMBER = new uint[] { 42000315, 0 };//On mac the version sent is 0 !?
```

Comment:
Are you sure about this? Are they really ignoring versions on mac clients?

Reply:
My 4.20 build send request version 0.
Thats for sure lol.

Reply:
If mac sends "Version 4.20.0.315 [PUBLIC]" and not the version number, you can actually remove that check at all. One should be enough. Although that can be done in another PR later. Your choice.